testsuite/cluster-test/src/experiments/mod.rs

```diff
@@ -109,10 +107,6 @@ pub fn get_experiment(name: &str, args: &[String], cluster: &Cluster) -> Box<dyn
     let mut known_experiments = HashMap::new();
 
     known_experiments.insert("recovery_time", f::<RecoveryTimeParams>());
-    known_experiments.insert(
```

Comment:
Why did we remove it?

Reply:
Will full node introduction, this experiment no longer correct because in its current form we inject latency into every outgoing packet of the validator - which means injecting latency b/w validator and its fullnode as well. 

This was effectively doing a 2 region simulation - if we need to run this simulation, we will have to rewrite it similarly to how we do the 3 region simulation where we selectively inject latencies b/w pairs of nodes.